README.md

```diff
@@ -269,6 +270,33 @@ or if you want to use base_ models and keep stg_model_2 as is:
 <img width="500" alt="A refactored DAG showing two base models feeding into a staging model" src="https://user-images.githubusercontent.com/30663534/159602135-926f2823-3683-4cd5-be00-c04c312ed42d.png">
 </details>
 
+#### Raw References
```

Comment:
Is it really raw reference? Or missing `source`/`ref`?

Reply:
Maybe hard-coded references?